ts-nestjs-sequelize/test/pulumi/index.ts

```diff
@@ -0,0 +1,29 @@
+import * as aws from "@pulumi/aws";
+import { PolicyPack, validateResourceOfType, StackValidationPolicy } from "@pulumi/policy";
+
+const stackPolicy: StackValidationPolicy = {
+    name: "redis-test",
```

Comment:
maybe typo? redis -> sequalize/rds

Reply:
this is just the lambda one so renamed. Will write utils/helpers tomorrow for all resources so we can reuse them.